app/src/test/java/org/mozilla/fenix/ext/ImageButtonTest.kt

```diff
@@ -0,0 +1,33 @@
+package org.mozilla.fenix.ext
+
+import kotlinx.coroutines.ObsoleteCoroutinesApi
+import org.mozilla.fenix.TestApplication
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import io.mockk.mockk
+import io.mockk.verify
+import android.widget.ImageButton
+
+@ObsoleteCoroutinesApi
+@RunWith(RobolectricTestRunner::class)
+@Config(application = TestApplication::class)
+
+class ImageButtonTest {
+    val mockIb: ImageButton = mockk(relaxed = true)
```

Comment:
nit: in Java/Kotlin convention, you generally avoid acronyms. I'd write `mockImageButton`. To be honest, since this is an image button test, `imageButton` is also appropriate. I haven't found the `mock` or `spy` prefixes to be all that useful and they're redundant to the type system, which already prevents you from making the wrong calls.